concent_api/core/tests/test_integration_report_computed_task.py

```diff
@@ -852,10 +984,22 @@ def test_requestor_sends_ack_report_computed_task_and_then_sends_reject_report_c
         self.assertEqual(response_3.status_code,  202)
         self.assertEqual(len(response_3.content), 0)
         self._test_database_objects(
-            last_object_type            = message.concents.AckReportComputedTask,
+            last_object_type            = message.concents.ForceReportComputedTask,
```

Comment:
`last_object_type` sounds suspicious. How does function determine which object in the database is "last"? If it just uses `last()` without enforcing the order with `order_by()` then this test is non-deterministic and may fail randomly.

Reply:
That is exactly why I wanted 

> Verifiication of type, task_id and subtask_id in those newly created StoredMessages (this might be done in single utils function, by passing list of types, task_id and subtask_id should be the same for all of them).

But the issue is that we can't assure order of the messages created in single request, because they are created with the same `timestamp` because we use `freeze_time`.

This method `_test_database_objects()` is removed completely in step 3 which removes old logic in favor of method which checks last x messages created during last request.

Reply:
ok